torch/_torch_docs.py

```diff
@@ -10549,6 +10551,13 @@ def merge_dicts(*dicts):
         for CPU tensor types and the current CUDA device for CUDA tensor types.
     {requires_grad}
     {check_invariants}
+    is_coalesced (bool, optional): When``True``, the caller is
+        responsible for providing tensor indices that correspond to a
+        coalesced tensor.  If the :attr:`check_invariants` flag is
+        False, no error will be raised if the prerequisites are not
+        met and this will lead to silently incorrect results.
+        Default: None, to save on overhead the resulting Tensor will
+        have the coalesced flag set to ``False``.
```

Comment:
But then it sounds like None always means False. You just pointed out that None can also sometimes mean True.

Reply:
@cpuhrsch, sorry, something went wrong in applying your suggestion last night... Now fixed, please review.